configure.php

```diff
@@ -914,13 +922,12 @@ function getFileModificationHistory(): array {
 $mxml = $ac["OUTPUT_FILENAME"];
 
 /* TODO: For some reason libxml does not validate the RelaxNG schema unless reloading the document in full */
-$dom->save($mxml);
-$dom->load($mxml, $LIBXML_OPTS);
+dom_saveload( $dom );   // idempotent path
+$dom->save($mxml);      // non idempotent, historical path
 if ($dom->relaxNGValidate(RNG_SCHEMA_FILE)) {
     echo "done.\n";
     printf("\nAll good. Saving %s... ", basename($ac["OUTPUT_FILENAME"]));
-    flush();
-    $dom->save($mxml);
+    $dom->save($mxml); // save it again... does validations changes somethnig?
```

Comment:
No, this is useless actually, and could probably be removed in a different PR/commit immediately.

Reply:
I will do some some tests to be certain. before removing it in this PR.

Reply:
The files are identical, as expected. Removing the redundant save(), message changed a little.